test/system/map_test.rb

```diff
@@ -3,35 +3,33 @@
 # https://guides.rubyonrails.org/testing.html#implementing-a-system-test
 
 class MapTest < ApplicationSystemTestCase
-  def setup
-    Capybara.default_max_wait_time = 60
-  end
+  Capybara.default_max_wait_time = 60
 
-  test 'correct url hash for wiki map' do
-    visit '/map/chicago'
+  # test 'correct url hash for wiki map' do
+  #   visit '/map/chicago'
 
-    # check that the url hash is correct
-    assert_equal("#13/41.87/-87.64", page.evaluate_script("window.location.hash"))
+  #   # check that the url hash is correct
+  #   assert_equal("#13/41.87/-87.64", page.evaluate_script("window.location.hash"))
     
-  end
+  # end
 
-  test 'show map by hash location' do
-    visit '/map#9/-25/-13'
+  # test 'show map by hash location' do
+  #   visit '/map#9/-25/-13'
 
-    assert_equal(-25, page.evaluate_script("Math.round(map.getCenter().lat)"))
-    assert_equal(-13, page.evaluate_script("Math.round(map.getCenter().lng)"))
-    assert_equal(9, page.evaluate_script("map.getZoom()"))
+  #   assert_equal(-25, page.evaluate_script("Math.round(map.getCenter().lat)"))
```

Comment:
I have a long shot thought that maybe the calculation inside the assertion could be failing this...
Do you mind trying doing something like this for one of the assertion to see if it passes?  
```
lat = page.evaluate_script("Math.round(map.getCenter().lat)")
assert_equal(-25, lat)?
```